app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,10 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: (ScihistDigicoll::Env.lookup(:no_reply_email_address) || "from@example.com")
+  default from: ScihistDigicoll::Env.lookup!(:digital_email_address)
   layout 'mailer'
+
+
+  def hostname
+    ScihistDigicoll::Env.lookup!(:app_url_base)
+  end
```

Comment:
I'm confused, do we actually NEED this in more than one mailer? Why'd you add it to the new DerivativeStorageTypeAuditMailer in the first place? (To then immediately remove it). 

It's a bit weird to need this thing like this; the fixity check mailer did, for it's own baroque reasons, which we don't need to change now. But I want to make sure the new one really does before adding it there or raising this to the superclass to be used by everything and kind of encouraging doing things this way.

Reply:
Or you only use it for `Digital Collections Warning: #{hostname}`.  Yeah, maybe just use the :service_level Env key?  (which will be like "staging" or "production" usually). Or, really, `ScihistDigicoll::Env.lookup!(:app_url_base)` is simple enough that I don't think it needs to be encapsulate din a new method?

`ScihistDigicoll::Env.lookup!(:app_url_base)` is ALSO not actually going to return a "hostname", it's going to return something like `https://hostname`.  Calling it hostname is a bit misleading too. Don't necessarily need to fix a misleading name that already existed in code, but I get more cautious when we're boosting htis method to be in the superclass ApplicationMailer, and without any comments too. I don't think it really needs to be there.

Reply:
Eh, I just saw two identical methods in two subclasses and automatically thought -- eh, let's move it to the superclass. Don't feel strongly about it; moved the method back down; it's not really part of the scope of this PR and it's not necessary, as you say.

Reply:
Cool, I'm not sure either one really needs it, but it's not this PR that's adding it, so I guess it's not this PR's problem! Thanks.